kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KieServerTest.java

```diff
@@ -255,9 +257,12 @@ public void testPrometheusEndpoint() {
 
     @Test
     public void testCommandServiceSetup() {
+        List<String> extensionsWithoutCommandSupport = asList(
+                PrometheusKieServerExtension.EXTENSION_NAME,
+                ScenarioSimulationKieServerExtension.EXTENSION_NAME);
         for (KieServerExtension extension : ((KieServerImpl) kieServer).getServerExtensions()) {
             KieContainerCommandService<?> tmp = extension.getAppComponents(KieContainerCommandService.class);
```

Comment:
A small suggestion to rename tmp on smth concrete)

Reply:
Renamed to `kieContainerCommandService`

Reply:
Cool thanks!